src/Microsoft.OData.Core/JsonLight/JsonLightMetadataLevel.cs

```diff
@@ -24,10 +24,25 @@ internal abstract class JsonLightMetadataLevel
         /// </summary>
         /// <param name="mediaType">The full media type being written. This media type must have a type/subtype of "application/json".</param>
         /// <param name="metadataDocumentUri">The metadata document uri from the writer settings.</param>
+        /// <param name="alwaysAddTypeAnnotationsForDerivedTypes">When set, type annotations will be added for derived types, even when the metadata level is set to "None".</param>

```

Comment:
>alwaysAddTypeAnnotationsForDerivedTypes [](start = 25, length = 39)

should remove this param

Reply:
Actually, the params don't match the actual params at all -- model and writingResponse are missing.

---
In reply to: [542754406](https://github.com/OData/odata.net/pull/1946#discussion_r542754406) [](ancestors = 542754406)

Reply:
Both `model` and `writingResponse` appear to have appropriate docs:
https://github.com/OData/odata.net/blob/3eeeff314fcb36b72c4c9be47dbb44f108fcebbe/src/Microsoft.OData.Core/JsonLight/JsonLightMetadataLevel.cs#L28

Reply:
Updated to remove incorrect `alwaysAddTypeAnnotationsForDerivedTypes` doc comment.